src/modules/dashboard/components/Dialogs/AddExistingSafeDialog/AddExistingSafeDialogForm.tsx

```diff
@@ -119,12 +129,34 @@ const AddExistingSafeDialogForm = ({
   // When selected chain or address changes
   useEffect(() => {
     // If there's a valid address in the address input
-    if (addressTouched && !addressError) {
-      // Get safe data
-      setIsLoadingSafe(true);
-      getSafeData(`${baseURL}api/v1/safes/${address}/`);
+    if (addressTouched && !addressError && address.length !== 0) {
```

Comment:
Crashing error here:
![Screenshot from 2022-07-26 15-27-20](https://user-images.githubusercontent.com/18473896/181083707-8518823d-c664-4e89-88fc-82557d132ff9.png)

This happens when I focus in and out of the address field

A better alternative to this could be `isAddress(address)`, that way you are covering `undefined`, `null`, `''`, `' '` and checking that the string is actually a valid address. The current implementation is only covering the `''` case.

Reply:
@ArmandoGraterol Good suggestion 👍